### DISCUSSION TITLE
Difficulty Adjust Floors and Ceilings

### DISCUSSION THREAD

**Discussion by KobeniCar** [ppy/osu#15388] [Ideas]
Set the Floor and Ceiling on a parameter in Difficulty Adjust, instead of a static value.

![theidea](https://user-images.githubusercontent.com/93515249/139645449-3251c10f-af93-43a2-a977-46d940eabab7.png)

For example, if a player sets the Floor for Circle Size to CS5, a beatmap at CS4 would be boosted to CS5. They could also set the Ceiling to be CS6, meaning a CS7 map would be lowered to that value. If CS is inbetween the Floor and Ceiling, it goes untouched.

**Comment by bdach** [comment]
How would the user then know what value a given map will use if they're just declaring ranges?

**Reply by KobeniCar** [reply]
Beatmap details would display whatever the Floor and Ceiling sets it to like how it does when its Static
